org.mintguide.WebAppReader.yaml

```
@@ -0,0 +1,62 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/bin/webappreader.sh
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home  # to be able to save html pages to a hard drive
+  - --share=network # it is necessary to download html from the Internet
+  #- --device=dri # For the Qt webengine module to work perfectly with gpu
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess # necessary for QtWebEngineProcess to work
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - python3-requirements.json
+
+  - name: wget2 #  proper workable version = 2.1
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap
+      - ./configure --prefix=/app
+      - make
+      - make install
+    sources:
+      - type: git
+        url: https://gitlab.com/gnuwget/wget2.git
+        commit: 657e3d22c488a2d1f173e208e253995e15f8b6db
+
+  - name: webappreader
+    buildsystem: simple
+    build-commands:
+      - install -D /dev/null /app/share/applications/org.mintguide.WebAppReader.desktop
+      - |
+        echo '[Desktop Entry]' > /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Name=WebAppReader' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Comment=Offline web content reader' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Exec=/app/bin/webappreader.sh %U' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Icon=org.mintguide.WebAppReader' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Terminal=false' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Type=Application' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+        echo 'Categories=Utility;' >> /app/share/applications/org.mintguide.WebAppReader.desktop
+      - 'echo "#!/bin/sh" > /app/bin/webappreader.sh'
+      - 'echo "python3 /app/webappreader.py \"$@\"" >> /app/bin/webappreader.sh'
+      - chmod +x /app/bin/webappreader.sh
+      - cp -rT /run/build/webappreader/database /app/database
+      - cp -rT /run/build/webappreader/pages /app/pages
+      - cp -rT /run/build/webappreader/src /app/src
+      - cp -rT /run/build/webappreader/ui /app/ui
+      - cp /run/build/webappreader/constants.py /app
+      - cp /run/build/webappreader/settings.py /app
+      - cp /run/build/webappreader/tools.py /app
+      - cp /run/build/webappreader/webappreader.py /app
```

```suggestion
      - cp -rT database /app/database
      - cp -rT pages /app/pages
      - cp -rT src /app/src
      - cp -rT ui /app/ui
      - install -Dm644 constants.py -t /app
      - install -Dm644 settings.py -t /app
      - install -Dm644 tools.py -t /app
      - install -Dm644 webappreader.py -t /app
```

note that installing directly in `/app` doesn't look right.